### PR TITLE
SVGLoader:  Update vertices according uv.y when linecap is square

### LINKED ISSUE #26781: SVGLoader pointsToStroke incorrect result using linecap square
### Description

When passing a simple 2 point line shape to SVGLoader pointToStroke, when linecap is set to square, the resulting geometry is incorrect.  

image

It works correctly when linecap is either butt or round.

### Reproduction steps

create a 2 point line shape
getStrokeStyle passing linecap set to 'square'
pass resulting shape and stroke style to pointsToStroke

### Code

```js
const shape = new Shape();
  shape.moveTo(-12, 0);
  shape.lineTo(12, 0);

  const style = SVGLoader.getStrokeStyle(
    2,
    "white",
    undefined,
    "square",
    undefined
  );

  const geometry = SVGLoader.pointsToStroke(shape.getPoints(), style);
  const material = new MeshBasicMaterial({ side: DoubleSide });
  const rect = new Mesh(geometry, material);
  scene.add(rect);
```

### Live example

https://codesandbox.io/s/three-stroke-square-test-3yn756

### Screenshots

image

### Version

156.1

### Device

Desktop

### Browser

Edge

### OS

Windows

### PR DESCRIPTION
**Pull request by nieyuyao**
fix #26781

### CODE REVIEW

File: examples/jsm/loaders/SVGLoader.js [hunk 1]
```diff
@@ -3082,7 +3082,8 @@ class SVGLoader extends Loader {
 						} else {
 
 							tempV2_3.toArray( vertices, 1 * 3 );
-							tempV2_3.toArray( vertices, 3 * 3 );
+							// using tempV2_4 to update 3rd vertex if the uv.y of 3rd vertex is 1
```
Comment (Mugen87) [inline examples/jsm/loaders/SVGLoader.js:3085]: The PR indeed fixes the issue and I don't see any regressions in `webgl_loader_svg`.

Do you mind explaining the reasoning behind this change though? Why does the code have to make a distinction at this line?

Comment (nieyuyao) [reply examples/jsm/loaders/SVGLoader.js:3085]: `tempV2_3` and `tempV2_4` are the two new vertices after the cap is calculated. They should replace the corresponding old vertices. Updates to  `tempV2_3` and `tempV2_4` are explicit. But there are two cases of updating vertices[3] with `tempV2_3` or `tempV2_4`.

image

`uv.y` is used to determine whether to use `tempV2_3` or `tempV2_4`.
image

Comment (nieyuyao) [reply examples/jsm/loaders/SVGLoader.js:3085]: It would be better if there was no need to make a distinction. Maybe there's a better way?

Comment (Mugen87) [reply examples/jsm/loaders/SVGLoader.js:3085]: Not sure. If somebody comes up with a uniform approach, we can sill update the code. For now, let's go with the this PR.

**Pull request merged**